tutor/specs/components/__snapshots__/lms-info-card.spec.jsx.snap

```diff
@@ -8,16 +8,18 @@ exports[`LmsInfo Component matches snapshot 1`] = `
 
 .c0 {
   margin: 10px 0 20px 0;
-  display: block;
+  padding-left: 0 !important;
```

Comment:
😲

Reply:
yeah.. it's overriding a bootstrap default 😭